apps/dashboard/src/app/account/account-users/account-users.component.ts

```diff
@@ -83,50 +80,41 @@ export class AccountUsersComponent implements OnDestroy {
         .pipe(switchMap(() => this.updateUsers()))
         .subscribe();
     } else {
-      const data: ConfirmData = {
-        title: 'generic.alert',
-        message: 'account.admin.warning',
-        confirmText: 'account.admin.make_admin',
-        minWidthButtons: '110px',
-      };
-      const dialogRef = this.dialog.open(STFConfirmComponent, {
-        width: '420px',
-        data: data,
-      });
-      dialogRef
-        .afterClosed()
-        .pipe(
-          takeUntil(this.unsubscribeAll),
+      this.modalService
```

Comment:
When using this modal an error of type "ExpressionChangedAfterItHasBeenCheckedError" appears in the console.

Reply:
good catch! let me fix that

Reply:
ok, I'm not able to find where the problem is coming from right now, so I'll open a ticket to solve this. As you said, this is not super urgent to fix